sway-core/src/asm_lang/virtual_immediate.rs

```diff
@@ -11,6 +11,8 @@ pub enum WideOperations {
     Add = 0,
     Sub = 1,
     Not = 2,
+    Lsh = 6,
```

Comment:
I assume there's an explanation for the jump in value, but what is it?

Reply:
This is coming from: http://specs.fuel.network/master/fuel-vm/instruction-set.html#wdop-misc-128-bit-integer-operations